.nais/deploy-catbox.yml

```diff
@@ -55,8 +55,18 @@ spec:
     outbound:
       rules:
         - application: pensjonskalkulator-backend
+      external:
+        - host: pensjonskalkulator-unleash-api.nav.cloud.nais.io
```

Comment:
I dag henter frontend feature-toggles via backend.
Betyr denne endringen at alle frontend-toggles kan hentes direkte fra Unleash?

Reply:
I teorien så kan det lages endepunkt for det, men trengte unleash for server for å hente redirect toggle, siden den gjøres på BFF siden nå.